components/Tabs/src/private/MDCItemBarBadge.m

```diff
@@ -36,7 +37,7 @@ - (instancetype)initWithFrame:(CGRect)frame {
 
 - (void)commonMDCTabBarItemBadgeInit {
   if (!_badgeColor) {
-    _badgeColor = [UIColor redColor];
+    _badgeColor = MDCPalette.redPalette.tint700;
```

Comment:
I know it's bad to use MDC color scheme defaults as default colors in components. I guess it's okay to do that with MDCPalette colors?

Reply:
Using the Material Palette colors for defaults is very often done in our components. We currently do this in:

* **ActivityIndicator** - Default cycle colors
* **NavigationDrawer** - Default handle color
* **ProgressView** - Default indicator color
* **Slider** - Default track color
* **TextFields** - Default error color

And more.

Reply:
I wasn't sure if it was still considered best practice. SGTM.

Reply:
I believe when we have to choose default colors, we prefer Material palette colors.